.github/workflows/release.yaml

```diff
@@ -124,50 +113,103 @@ jobs:
           draft: true
           prerelease: false
           fail_on_unmatched_files: true
-          files: |
-            ${{ env.ASSET }}
-            ${{ env.ASSET }}.sha256
-            ${{ env.ASSET }}.sha512
+          files: "${{ steps.build.outputs.files }}"
 
-  build-release-wasm:
-    name: build-release-wasm
+      - name: Upload artifact
+        uses: actions/upload-artifact@v4
+        with:
+          name: release-${{ matrix.target }}
+          path: "${{ steps.build.outputs.files }}"
+          overwrite: true
+
+  build-container-images:
+    name: build-container-images
+    needs: [ build-release ]
     runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'push' || (github.repository == 'gleam-lang/gleam' && needs.prepare-nightly.outputs.should-run != 'false') }}
+    strategy:
+      matrix:
+        base-image:
+          - scratch
+          - erlang
+          - erlang-slim
+          - erlang-alpine
+          - elixir
+          - elixir-slim
+          - elixir-alpine
+          - node
+          - node-slim
+          - node-alpine
```

Comment:
Currently containers are not created until after the release is published, but if I'm reading this right the containers would be published _before_ the release is, and would still be released even if the release was rejected.

Reply:
That's still not the behaviour from before unfortunately. When the release workflow finishes it leaves a GH release in draft mode. I can then check this release, the assets, and anything else before publishing it, and publication then results in the container images being built and published. With this new workflow the container images would be published even if I reject the release and do not publish it.

Reply:
Ah, I misunderstood how that worked before. Yes, it now works differently.
I'll take the PR back into draft state and fix this.